[2024-04-13 18:29:59,934] INFO _log() L224   * Restarting with watchdog (windowsapi) - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:12,986] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\sqlalchemy\\sql\\_orm_types.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:14,136] INFO _log() L224   * Restarting with watchdog (windowsapi) - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:21,317] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:21,319] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:21,320] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:22,490] INFO _log() L224   * Restarting with watchdog (windowsapi) - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:30:38,340] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:30:38] "[33mDELETE /estado/1 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:08,989] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\sqlalchemy\\orm\\query.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:08,999] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:09,003] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:19,116] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.157:5000 - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:19,117] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:19,491] INFO _log() L224   * Restarting with watchdog (windowsapi) - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:24,322] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:31:24] "[33mDELETE /estado/1 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:30,454] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:31:30] "[33mDELETE /estado/id:1 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:34,295] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:31:34] "[33mDELETE /estado/id:2 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:38,438] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:31:38] "[33mDELETE /estado/2 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:31:53,662] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:31:53] "[33mDELETE /estado/2 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:32:00,704] WARNING add_estado() L61   Erro ao adicionar Estado 'alexa21ndre', Estado de mesmo nome já salvo na base :/ - call_trace=C:\Users\sp-xa\OneDrive\Área de Trabalho\api_python\app.py L61  
[2024-04-13 18:32:00,705] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:32:00] "[31m[1mPOST /estado HTTP/1.1[0m" 409 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:32:04,334] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:32:04] "GET /estados HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:32:09,258] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:32:09] "[33mDELETE /estado/2 HTTP/1.1[0m" 404 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:32:53,313] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:32:53] "DELETE /estado?id=2 HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:33:52,216] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:33:52] "GET /estados HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:33:55,898] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:33:55] "OPTIONS /estado?id=Rio%20das%20Pirocas HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:33:55,901] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:33:55] "[31m[1mDELETE /estado?id=Rio%20das%20Pirocas HTTP/1.1[0m" 422 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:46:07,167] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:46:07,170] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:46:07,171] INFO _log() L224   * Detected change in 'C:\\Users\\sp-xa\\OneDrive\\Área de Trabalho\\api_python\\app.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:46:08,365] INFO _log() L224   * Restarting with watchdog (windowsapi) - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:47:16,585] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:47:16] "GET /estados HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:47:49,242] WARNING add_estado() L61   Erro ao adicionar Estado 'alexa21ndre', Estado de mesmo nome já salvo na base :/ - call_trace=C:\Users\sp-xa\OneDrive\Área de Trabalho\api_python\app.py L61  
[2024-04-13 18:47:49,243] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:47:49] "[31m[1mPOST /estado HTTP/1.1[0m" 409 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:47:52,388] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:47:52] "[31m[1mPOST /estados HTTP/1.1[0m" 405 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:47:57,958] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:47:57] "[31m[1mPOST /estados HTTP/1.1[0m" 405 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:48:14,177] WARNING add_estado() L61   Erro ao adicionar Estado 'alexa21ndre', Estado de mesmo nome já salvo na base :/ - call_trace=C:\Users\sp-xa\OneDrive\Área de Trabalho\api_python\app.py L61  
[2024-04-13 18:48:14,178] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:48:14] "[31m[1mPOST /estado HTTP/1.1[0m" 409 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:48:18,273] INFO _log() L224  127.0.0.1 - - [13/Apr/2024 18:48:18] "GET /estados HTTP/1.1" 200 - - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:45,800] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\flask_openapi3\\scaffold.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:45,958] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\capture.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:46,007] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\doctest.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:46,039] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\compat.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:46,084] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\junitxml.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:46,120] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\legacypath.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2024-04-13 18:49:46,166] INFO _log() L224   * Detected change in 'C:\\ProgramData\\anaconda3\\Lib\\site-packages\\_pytest\\logging.py', reloading - call_trace=C:\ProgramData\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
